Tribler/Core/Utilities/utilities.py

```diff
@@ -1,7 +1,10 @@
+"""
+This class mainly provides validation and correction for both metainfo and urls. This class
```

Comment:
This is [PEP 257](http://legacy.python.org/dev/peps/pep-0257/#documentation-strings):

> The docstring for a module should generally list the classes, exceptions and functions (and any other objects) that are exported by the module, with a one-line summary of each. (These summaries generally give less detail than the summary line in the object's docstring.)